third_party/tensorrt/LICENSE

```diff
@@ -0,0 +1,203 @@
+Copyright 2018 The TensorFlow Authors.  All rights reserved.
```

Comment:
I'll do some change internally to add this file since it conflicts with some internal config, and remove it from this PR. Just FYI.

Reply:
@aaroey, sure go ahead.